R/methods.R

```diff
@@ -962,6 +957,14 @@ plot.vsel <- function(
   }
 
   # Create the plot:
+  if (deltas) {
+    data_gg$statistic[data_gg$statistic=="elpd"] <- "elpd_diff"
+    stats_ref$statistic[stats_ref$statistic=="elpd"] <- "elpd_diff"
+    data_gg$statistic[data_gg$statistic=="mlpd"] <- "mlpd_diff"
+    stats_ref$statistic[stats_ref$statistic=="mlpd"] <- "mlpd_diff"
+    data_gg$statistic[data_gg$statistic=="gmpd"] <- "gmpd_ratio"
+    stats_ref$statistic[stats_ref$statistic=="gmpd"] <- "gmpd_ratio"
```

Comment:
Aren't several performance statistics missing here? (mse, rmse, acc, pctcorr, auc)

Reply:
Ah, forgot to tell that, after some discussion with workflow group members, I changed it so that when `deltas=TRUE` only these `**pd` criteria show diff, and all others still use the original scale. Using delta helps to see the uncertainty compared to the reference model, but using the original scale is more clear for all other criteria. I understand there is possibility of confusion, but I really prefer the plots with the original scale. Here is an example plot
![image](https://github.com/stan-dev/projpred/assets/6705400/be287633-c675-46f6-8285-411190a87ef1)

I need to fix the documentation about this

Reply:
Ok. We could also think about a warning for those performance statistics where `deltas = TRUE` will still use the original scale (with the possibility to suppress that warning via a global option).

Reply:
Added, and used the global option `projpred.extra_verbose`

Reply:
Thanks. But it looks like you added this only in `plot.vsel()` (at least `summary.vsel()` should also be concerned, I guess). Furthermore, with `projpred.extra_verbose`, this is an "opt-in" warning/message because users need to set `projpred.extra_verbose` to `TRUE` in order to get this message. I would prefer an "opt-out" solution (e.g., adding a new global option such as `projpred.warn_deltas` which is by default `TRUE`) because this is such an important change.

Reply:
Thinking about the whole `deltas` change again, I think I would personally avoid it (i.e., revert back to the former behavior, meaning that for all performance statistics, `deltas = TRUE` will refer to the difference vs. the baseline model, no matter whether with respect to the point estimate or the uncertainty) because I think this change will be confusing for many users and because it's easy to overlook this change. This is mitigated if an "opt-out" warning/message is thrown, but I still find it a quite complicated change.

Nevertheless, if you prefer to keep this change, I won't object to it. I'm seeing myself as an "advisor" here, not as the person making the design decisions.

Reply:
I just read your comment again, in particular the part

> I understand there is possibility of confusion, but I really prefer the plots with the original scale.

Even with respect to the plots, I think that the former behavior of `deltas = TRUE` is preferable. Users can always create a separate plot with `deltas = FALSE` and (if desired) combine plots via the **patchwork** package, for example. That way, we can be sure that users are aware of what they are doing.

I understand your comment

> using the original scale is more clear for all other criteria

but for this, we could also throw a warning or a message if users request `deltas = TRUE` for statistics other than the `**pd` ones.

Reply:
Ah, I now understand your point: Above, you didn't mention that the `deltas = TRUE` uncertainty for non-`**pd` statistics would still refer to the difference vs. the baseline model. So this is indeed a new type of plot / summary. But then, wouldn't it be best to add a new argument for this new type of plot / summary? Or allow `deltas` to be set to something like `"combined"` (or `NULL` or `NA`) to use this new type? That way, `deltas = FALSE` and `deltas = TRUE` could still behave as before (which would be good for backwards compatibility and to avoid confusion among users), but users would also have the option to use the new type.

Reply:
Marking this conversation as "resolved" because this problem has been circumvented (for this PR) by moving the new "mixed deltas" plot, the new column behavior of `summary.vsel()`, and some other changes to branch [`mixed_deltas_plot`](https://github.com/stan-dev/projpred/tree/mixed_deltas_plot). In the PR for that `mixed_deltas_plot` branch, this needs to be taken into account.